torch/csrc/jit/export.cpp

```diff
@@ -99,6 +100,7 @@ class EncoderBase {
 
   size_t num_blocks_;
   bool defer_weight_export_;
+  bool pretty_print_;
```

Comment:
Is the only thing that this controls is stripping docs? Can we name it something like `strip_docs_` if that's the case

Reply:
yes, it is. Yap, sounds good. I will update it.